### ISSUE TITLE
Beta and SNAPSHOT installation instructions missing

### ISSUE THREAD

**Issue by tobrun** [mapbox/android-docs#118]
The previous version of the documentation showed installation instructions for beta and snapshots builds. This information is not listed anywhere else + all older issues referring to this page to check for this information don't make sense anymore. 

cc @cammace @langsmith @zugaldia

**Comment by cammace** [comment]
This goes along with https://github.com/mapbox/android-docs/issues/58, If we see an immediate need to add toggles to the documentation I can add them (though it's probably not easy to setup). For SNAPSHOTs, all other documentation examples I've looked at place this information inside the readme file instead of directly inside the documentation.

**Comment by zugaldia** [comment]
👍 While we can discuss the best placement for SNAPSHOT installation instructions, we definitely need a toggle for betas.

**Comment by cammace** [comment]
Cool, will look into adding later this week.

**Comment by langsmith** [comment]
@colleenmcginnis , can you and I chat about toggles soon? I hope to learn so that I can ✔️ this issue and for preparation ahead of Java/Kotlin fun

**Comment by colleenmcginnis** [comment]
Yes yes yes. Does late next week work? I could set up the initial infrastructure by Wednesday EOD and then we could talk about contributing.
